prep-example/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>prep-example</title>
+</head>
+<body>
+  <h1>I would like to build</h1>
+  <p>
+    <!-- write your app ideas here 👇 -->
+
```

Comment:
It looks like possibly your HTML file was not saved before you did the `git add` and `git commit` steps. That's ok! we have steps on how to fix it.

https://lfz-lms.herokuapp.com/code-guides/Learning-Fuze/curriculum/pull-request-feedback